i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/fragments/_api_resource_schema.mdx

```diff
@@ -34,7 +34,7 @@
         令牌过期时间 <i>（秒）</i>
```

Comment:
nit: no need to add whitespace between the Chinese character the full-width punctuation

Reply:
i think it's because of the italic font. i'll merge this one first. let simeng know if you have any concerns, we can update later.